cachito/web/metrics.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+from prometheus_client import multiprocess
+from prometheus_client.core import CollectorRegistry
+from prometheus_flask_exporter import PrometheusMetrics
+
+
+def init_metrics():
+    """
+    Initialize the Prometheus Flask Exporter.
+
+    :return: a Prometheus Flash Metrics object
+    :rtype: PrometheusMetrics
+    """
+    registry = CollectorRegistry()
+    if "PROMETHEUS_MULTIPROC_DIR" not in os.environ:
+        raise Exception(
+            "Temp directory for prometheus-flask metrics not defined in environment variable: \
+            PROMETHEUS_MULTIPROC_DIR"
+        )
```

Comment:
```suggestion
        raise Exception(
            "Temp directory for prometheus-flask metrics not defined in environment variable: "
            "PROMETHEUS_MULTIPROC_DIR"
        )
```

Reply:
Thanks, didn't realize it was that simple....